Project 1/styles.css

```diff
@@ -0,0 +1,20 @@
+.card{
+        max-width: 800px;
+        margin: 30px auto;
+        background-color: #ffffff;}
+
+img{
+    max-width: 100%;}
+    title {
+        font-family: 'Slabo', serif;
```

Comment:
You reference both Slabo and Roboto fonts, but do not add those webfonts, so most users will not see them.